editor/package.json

```diff
@@ -444,7 +444,6 @@
       "eslint-plugin-jsx-a11y@6.5.1": "patches/eslint-plugin-jsx-a11y@6.5.1.patch",
       "karma-mocha-reporter@2.2.5": "patches/karma-mocha-reporter@2.2.5.patch",
       "keycode@2.2.1": "patches/keycode@2.2.1.patch",
-      "react-contexify@5.0.0": "patches/react-contexify@5.0.0.patch",
```

Comment:
As noted in PR description, let me know if (and how) this needs to be re-created for v6.